### ISSUE TITLE
Encrypt local fight-covid-db using SQLCipher

### ISSUE THREAD

**Issue by sudheesh001** [nic-delhi/AarogyaSetu_Android#25]
The current android room database present locally doesn't encrypt and store the information even locally which would be good security practice to do. This can be easily implemented with `SQLCipher` where the plaintext data like `latitude, longitude, bluetooth_mac_address, timestamp` are encrypted on the local device using a randomly generated symmetric key from the `AndroidKeyStore` for which this app already seems to contains support taking a look at the SecureUtils

**Comment by Lav-Hinsu** [comment]
someone should implement this and make a pull request. with the influx of issues coming now, i don't think anyone from the devs is going to read this till tomorrow. this seems like a serious issue to me

**Comment by SagarSDagdu** [comment]
I don't think this is needed as the data that is being saved in these fields is already encrypted. Have a look at the methods of saving latitude and longitude here : 
https://github.com/nic-delhi/AarogyaSetu_Android/blob/8b96cb981426297028ea7e0d78f0aaee325f3ee7/app/src/main/java/nic/goi/aarogyasetu/models/BluetoothData.java#L94

**Comment by sudheesh001** [comment]
Yes, I see the latitude and longitude being encrypted in these fields but wouldn't it just be easier and cleaner in the code to process this information in plaintext with the entire database being encrypted than the individual fields? 

At the end of the day in the `upload()` procedures, this information is decrypted before being sent to the server, so I don't see a reason to roll your own crypto here of using AES to encrypt them especially when the mode being used in older devices with ECB is insecure to chosen plaintext attacks. #51
